packages/core-components/src/components/EmptyState/MissingAnnotationEmptyState.tsx

```diff
@@ -53,16 +53,21 @@ const useStyles = makeStyles<BackstageTheme>(
   { name: 'BackstageMissingAnnotationEmptyState' },
 );
 
-export function MissingAnnotationEmptyState(props: Props) {
-  const { annotation, readMoreUrl } = props;
+export function MissingAnnotationEmptyState({
+  annotation,
+  readMoreUrl,
+}: Props) {
   const url =
     readMoreUrl ||
     'https://backstage.io/docs/features/software-catalog/well-known-annotations';
   const classes = useStyles();
+  const annotationsToDisplay = Array.isArray(annotation)
+    ? annotation
+    : [annotation];
   const description = (
     <>
-      The <code>{annotation}</code> annotation is missing. You need to add the
-      annotation to your component if you want to enable this tool.
+      The <code>{annotation.toString()}</code> annotation is missing. You need
```

Comment:
Probably not informative enough, this won't look good. You should probably change the code to look for all annotations, and keep track of the subset of them that actually didn't exist, and then show this text as a more human friendly enumeration of them. Maybe with commas and stuff, and plural-"s" if more than one etc.